app.py

```diff
@@ -310,6 +316,51 @@ def save_identity(caller_identity):
     return jsonify({})
 
 
+# End Point which creates payout info next to identity
+@app.route('/v1/identities/<identity_url_param>/payout', methods=['POST'])
+@validate_json
+@recover_identity
+def create_payout_info(identity_url_param, caller_identity):
+    payload = request.get_json(force=True)
+
+    identity_payload = payload.get('identity', None)
+    if identity_payload is None:
+        msg = 'missing identity parameter in body'
+        return jsonify(error=msg), 400
+
+    payout_eth_address = payload.get('payout_eth_address', None)
+    if payout_eth_address is None:
+        msg = 'missing payout_eth_address parameter in body'
+        return jsonify(error=msg), 400
+
+    if identity_payload.lower() != identity_url_param.lower():
+        msg = 'identity parameter in url does not match with provider_id'
+        return jsonify(error=msg), 400
+
+    if identity_payload.lower() != caller_identity:
+        msg = 'identity parameter in body does not match with signer identity'
+        return jsonify(error=msg), 400
+
+    if not is_valid_eth_address(payout_eth_address):
+        msg = 'payout_eth_address is not in Ethereum address format'
+        return jsonify(error=msg), 400
+
+    if IdentityRegistration.query.get(caller_identity):
+        msg = 'identity payout address already registered'
+        return jsonify(error=msg), 403
```

Comment:
I think we might need to allow to override the address. I might want to change it.

Reply:
Actually, @AndriusMyst prepared draft prototype - it does include ability to change the address :)

Reply:
ability to change address implemented